votor/src/event_handler.rs

```diff
@@ -388,10 +414,10 @@ impl EventHandler {
         parent_block @ (parent_slot, parent_block_id): Block,
```

Comment:
we should touch up the function comment here to indicate what the returned `Result` and `bool` mean